pkg/tests/upgrade_tests/shapshot.go

```diff
@@ -184,6 +231,7 @@ func SnapshotDB(t *testing.T, container testcontainers.Container, dbName, output
 		info.triggers = getPsqlInfo(t, container, dbName, outputDir, "\\dy "+schema+".*")
 		info.data = getTableInfosForSchema(t, db, schema)
 	}
+	info = ClearTableFromSnapshot(info, "_ps_catalog", "migration")
```

Comment:
We should put in a TODO or issue to handle this more carefully in the future. We do want some checks on this table.

Reply:
https://github.com/timescale/promscale/issues/1237